data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditableDataResponse.java

```diff
@@ -0,0 +1,151 @@
+    public String getToken() {
+        return token;
+    }
+
+    @Nullable
+    @Generated
+    public AuditSuccessMessage getAuditSuccessMessage() {
+        return auditSuccessMessage;
+    }
+
+    @Nullable
+    @Generated
+    public AuditErrorMessage getAuditErrorMessage() {
+        return auditErrorMessage;
+    }
+
+    /**
+     * The static builder
+     */
+    public static class Builder {
+
+        /**
+         * The starter method for the Builder class.
+         * *
+         * @return the composed immutable object
+         */
+        public static IToken create() {
+            return  token ->  success -> error ->  new AuditableDataResponse(token, success, error);
+        }
+
+        /**
+         * Adds the token to the message
+         */
+        public interface IToken {
+            /**
+             * Adds the token to the request
+             *
+             * @param token the client's unique token
+             * @return interface {@link ISuccess} for the next step in the build.
+             */
+            ISuccess withToken(String token);
+        }
+
+        /**
+         * Adds the auditSuccessMessage for the message.
+         */
+        public interface ISuccess {
+            /**
+             * Adds the su for the message.
+             *
+             * @param auditSuccessMessage timestamp for the request.
+             * @return interface {@link IError} for the next step in the build.
+             */
+            IError withSuccessMessage(AuditSuccessMessage auditSuccessMessage);
+
+        }
+
+        /**
+         * Adds the attributes for the message.
+         */
+        public interface IError {
+            /**
+             * Adds the attributes for the message.
+             *
+             * @param auditErrorMessage timestamp for the request.
+             * @return interface {@link AuditableDataResponse} for the final step in the build.
+             */
+            AuditableDataResponse withAuditErrorMessage(AuditErrorMessage auditErrorMessage);
+        }
+
+    }
+
+    @Override
+    @Generated
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof AuditableDataResponse)) {
+            return false;
+        }
+        AuditableDataResponse that = (AuditableDataResponse) o;
+        return Objects.equals(auditSuccessMessage, that.auditSuccessMessage) &&
+                Objects.equals(auditErrorMessage, that.auditErrorMessage);
+    }
+
+    @Override
+    @Generated
+    public int hashCode() {
+        return Objects.hash(auditSuccessMessage, auditErrorMessage);
+    }
+
+    @Override
+    @Generated
+    public String toString() {
+        return new StringJoiner(", ", AuditableDataResponse.class.getSimpleName() + "[", "]")
+                .add("auditSuccessMessage=                " + auditSuccessMessage)
+                .add("auditErrorMessage=                " + auditErrorMessage)
```

Comment:
Remove the unwanted whitespace and is there a reason the token value is not being printed?
```suggestion
                .add("token=" + token)
                .add("auditSuccessMessage=" + auditSuccessMessage)
                .add("auditErrorMessage=" + auditErrorMessage)
```

Reply:
?? This is from the Palisade  toString.  Did it as part of the other changes

Reply:
Check your code gen xml files.
If they've been auto-indented, they're likely inserting that indentation as spaces in this string